### PR TITLE
Add SONiC VM template as a peer

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

File: ansible/roles/vm_set/templates/sonicvm.xml.j2 [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <driver name='qemu' type='qcow2' cache='writeback'/>
+      <source file='{{ disk_image }}'/>
+      <target bus='virtio' dev='vda'/>
+    </disk>
+    <interface type='bridge'>
+      <model type='virtio'/>
+      <source bridge='{{ mgmt_bridge }}'/>
+      <target dev='{{ mgmt_tap }}'/>
+    </interface>
+{% for fp_num in range(0, max_fp_num) %}
```
Comment (lguohan) [inline ansible/roles/vm_set/templates/sonicvm.xml.j2]: are we increasing the max_fp_num to 32?

**Comment by jipanyang** [comment]
What is the plan for existing sonic.xml.j2?   Any issue seen with it?

**Comment by lguohan** [comment]
it is better to unify them. this sonicvm.xml.j2 is used to create sonic eos VM as DUT peer, so that we can have sonic T1 to be peer of sonic dut.

**Comment by jipanyang** [comment]
Yes, this is a good change.  Also for T0 topology, hope to see the topology script update to accommodate this change soon.

**Pull request closed without merging**